gradle.properties

```diff
@@ -0,0 +1,2 @@
+group = org.threadly
+version = 3.3.0-SNAPSHOT
```

Comment:
I suggest we version this independently, starting at 1.0.0, unless you see a reason to have it matching the threadly version?